hapi/hedera-protobufs/block/stream/output/state_changes.proto

```diff
@@ -786,17 +797,9 @@ message QueuePushChange {
         google.protobuf.StringValue proto_string_element = 2;
 
         /**
-         * A transaction record entry added to queue state.
-         * <p>
-         * <blockquote>REVIEW NOTE<blockquote>
-         * Do we really intend to push `transaction records` as state
-         * changes in the block stream? I would think that any queue holding
-         * transaction records is temporary storage (e.g. record cache) and
-         * will be removed or replaced with a block item queue as part
-         * of the change over to block stream.
-         * </blockquote></blockquote>
+         * All the transaction receipts for a round added to queue state.
          */
-        proto.TransactionRecordEntry transaction_record_entry_element = 3;
+        proto.TransactionReceiptEntries transaction_receipt_entries_element = 3;
```

Comment:
The removed question is relevant.
Do we actually need these as state changes?  Can we elide this list and save the bytes in the block stream, or is it something that is actually required in state to maintain deterministic behavior?

Reply:
We need the last 180 consensus seconds of receipts in state for deterministic duplicate classification and fee charging.

There may be future optimizations to maintain this information out of state; but I think for now we need it.